auth-web/src/components/auth/TestimonialQuotes.vue

```diff
@@ -1,19 +1,47 @@
 <template>
   <section id="testimonials-container">
     <v-container class="quote-container">
-      <p>"We're incredibly excited to be able to incorporate as a Benefit Company in BC, it really reflects our values
-        and the role we want to play."</p>
-      <p><strong>&ndash; BC Benefit Company user</strong></p>
+      <div :class="{ 'fadeOut': isFading, 'fadeIn': !isFading}">
+        <p>{{quotes[quoteIndex].text}}</p>
+        <p><strong>&ndash; {{quotes[quoteIndex].author}}</strong></p>
+      </div>
     </v-container>
   </section>
 </template>
 
 <script lang="ts">
 import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
-import LearnMoreButton from '@/components/auth/common/LearnMoreButton.vue'
 
 @Component({})
-export default class TestimonialQuotes extends Vue {}
+export default class TestimonialQuotes extends Vue {
+  private isFading: boolean = false
```

Comment:
you can avoid the explicit typing for simple values.. TS can infer it... private isFading =.false will do

Reply:
I concur.
In fact, depending on TS flags, you get a (lint) error if you include explicit types where not needed.
My "litmus test" is: if I hover over the property, does it display its type? (not any!)

Reply:
Updated in next commit 👍